test/e2e/tests/request-queuing/dapp1-switch-dapp2-send.spec.js

```diff
@@ -240,37 +221,19 @@ describe('Request Queuing Dapp 1, Switch Tx -> Dapp 2 Send Tx', function () {
           `window.ethereum.request(${switchEthereumChainRequest})`,
         );
 
-        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
-        await driver.findElement({
-          text: 'Use your enabled networks',
-          tag: 'p',
-        });
-
         await driver.switchToWindowWithUrl(DAPP_ONE_URL);
 
         await driver.clickElement('#sendButton');
 
         await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
 
-        await driver.clickElement({ text: 'Cancel', tag: 'button' });
+        await driver.clickElement({ text: 'Confirm', tag: 'button' });
         await driver.switchToWindowWithUrl(DAPP_ONE_URL);
 
         // Wait for switch confirmation to close then tx confirmation to show.
         // There is an extra window appearing and disappearing
         // so we leave this delay until the issue is fixed (#27360)
         await driver.delay(5000);
-        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
-
-        // Check correct network on the send confirmation.
-        await driver.findElement({
-          css: '[data-testid="network-display"]',
-          text: 'Localhost 8546',
-        });
-
-        await driver.clickElementAndWaitForWindowToClose({
-          text: 'Confirm',
-          tag: 'button',
-        });
```

Comment:
this should stay as well. The goal here is to confirm that the network switch on Dapp One did not affect Dapp Two

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/27980/commits/7f7646f7a81ee1c998cfddde8a2185b9099fa80c